euphoria-hbase/src/main/java/cz/seznam/euphoria/hbase/HFilesSink.java

```diff
@@ -192,22 +212,72 @@ public boolean prepareDataset(
           }
         })
         .withSortedValues(comparator::compare)
+        .output();
+
+    String outputDir = HFilesSink.this.getConfiguration().get(FileOutputFormat.OUTDIR);
+
+    ReduceWindow.of(hfiles)
```

Comment:
Hm, making it optional would change a lot the underlaying logic. What should happen when the bulk load doesn't occur? The `ReduceWindow` is at the moment necessary, because `DataSink.commit()` is never called during the execution, so the resulting HFiles otherwise remain somewhere in `_temporary` directory. What I can do is make configurable the call to `loadIncrementalHFiles`, so that the hfiles would simply accumulate in the target directory. But the reducing part is necessary.